sw/device/silicon_creator/lib/manifest.h

```diff
@@ -494,14 +507,14 @@ typedef struct manifest_ext_isfb {
   /**
    * Strikeout mask.
    *
-   * The size of this array is equal to 128 strike bits packed into uint32_t
-   * words. Each strike bit corresponds to a flash word in the ISFB info flash
-   * page.
+   * The size of this array is equal to 256 strike bits packed into uint32_t
```

Comment:
Can we keep the number of strike out bits at 128 (that should be all we need). I would like to keep the other space free for now.

Layout of info page would be
 * 0 - 512: Owner firmware rollback constraint section (128 4-byte "bits")
 * 512- 1024: Unused and available to owner firmware
 * 1024 - (1024 + owner-defind limit): product expression constraints
 * (1024 + owner-defind limit) - 2048: Unused and available to owner firmware

Reply:
Updated to match suggested info page layout.